Sources/Danger/GitLabDSL.swift

```diff
@@ -53,7 +54,10 @@ extension GitLab {
             public let dueDate: Date?
             public let id: Int // swiftlint:disable:this identifier_name
             public let iid: Int
-            public let projectId: Int
+            /// Will has value if it is a regular [project milestone](https://docs.gitlab.com/ee/api/milestones.html). Or it will be nil.
+            public let projectId: Int?
+            /// Will has value if it is a [group milestone](https://docs.gitlab.com/ee/api/group_milestones.html). Or it will be nil.
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 12...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/GitLabDSL.swift;
  Line: 59;
-->

- :warning: Line should be 120 characters or less: currently 133 characters (`line_length`)